Makefile

```diff
@@ -50,17 +50,17 @@ test-integration:
 .PHONY: verify-gator
 verify-gator:
 ifeq ($(POLICY_ENGINE), rego)
-	gator verify ./... --experimental-enable-k8s-native-validation=false
+	gator verify ./... --enable-k8s-native-validation=false
```

Comment:
We need a conditional check here for `GATOR_VERSION`, for < 3.17 use `experimental` flag, for >= 3.17 use updated flag.